.github/support.yml

```diff
@@ -0,0 +1,14 @@
+# Configuration for support-requests - https://github.com/dessant/support-requests
+
+# Label used to mark issues as support requests
+supportLabel: support-request
+# Comment to post on issues marked as support requests. Add a link
+# to a support page, or set to `false` to disable
+supportComment: >
+  We use the issue tracker exclusively for bug reports and feature requests.
+  However, this issue appears to be a support request. Please use our
+  [Discourse](https://discourse.brew.sh) to get help with the project.
```

Comment:
Slightly concerned this is going to send people to Discourse who should actually be talking to upstream for the given software.

Reply:
I think that's fine, honestly. Discourse doesn't get pruned or clogged up and we've already got a culture there where it's much more acceptable for topics to go unanswered and users to help other users. Feel free to disagree, though.

Reply:
Is it worth mentioning the IRC channel here as well?